### PR TITLE
PUB-1634 Added functional tests for 'Unauthorized' status code

### PR DESCRIPTION
**Pull request by ashwini-mv**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PUB-1634

### Change description ###
Added functional tests for 'Unauthorized' status code

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotifyTest.java [hunk 1]
```diff
@@ -459,6 +487,18 @@ void testValidFlatFileRequestCsv() throws Exception {
             .andExpect(status().isOk());
     }
 
+    @Test
+    @WithMockUser(username = "unknown_user", authorities = {"APPROLE_api.request.unknown"})
+    void testUnauthorizedSendSubscriptionEmail() throws Exception {
+        String validBody =
```
Comment (KianKwa) [inline src/integrationTest/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotifyTest.java:493]: We have upgraded Java to version 17 in the project. So you can use the latest Java multi-line text block for json string. It is a bit more readable. You can see an example at the top of this file. Also can take a look at examples here:
https://mkyong.com/java/java-multi-line-string-text-blocks/

File: src/integrationTest/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotifyTest.java [hunk 2]
```diff
@@ -168,12 +168,12 @@ class NotifyTest {
         """;
     private static final String VALID_INACTIVE_USER_NOTIFICATION_EMAIL_BODY =
         """
-        {
-            "email": "test@test.com",
-            "fullName": "testName",
-            "lastSignedInDate": "01 May 2022"
-        }
-        """;
+            {
```
Comment (KianKwa) [inline src/integrationTest/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotifyTest.java:171]: Why is this bit indented now. For multi-line text block, think the text block should start at the beginning

File: src/integrationTest/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotifyTest.java (hunk 1 above)
Comment (ashwini-mv) [reply src/integrationTest/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotifyTest.java:493]: Fixed

File: src/integrationTest/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotifyTest.java (hunk 2 above)
Comment (ashwini-mv) [reply src/integrationTest/java/uk/gov/hmcts/reform/pip/publication/services/controllers/NotifyTest.java:171]: Fixed

**Review by joshblackmoor — APPROVED** [review]

**Review by KianKwa — APPROVED** [review]

**Pull request merged**